app/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -18,6 +18,7 @@ async function remoteSettings(arguments) {
 
     const context = TargetingContext.TargetingContext.combineContexts(
         _experiment,
+        {defaultProfile: {}}, // Workaround for supporting background tasks
```

Comment:
Can I rewrite the comment, or bulk out the commit comment?  Say something like:

In Firefox Desktop, the targeting context is not fixed.  From the perspective of Experimenter there is only one `firefox-desktop` application, but the client splits Nimbus features between two sub-application IDs: `firefox-desktop` and `firefox-desktop-background-task`.  The `firefox-desktop-background-task` sub-application applies to `firefox --backgroundtask ...` invocations.  Not all features apply to both sub-application IDs, and the targeting context for the `firefox-desktop-background-task` sub-application ID includes details about the default browsing profile not present to the regular `firefox-desktop` ID.  Ideally, this `defaultProfile` would only appear when the test experiment's feature applies to `firefox-desktop-background-task`, but since the test experiments created here use a constant feature, this isn't possible at this time.

Reply:
Yeah we use squash commits so we can add that information to the commit at squash time.